src/solver/conv_asm_1x1u_bias_activ_fused.cpp

```diff
@@ -92,35 +92,13 @@ PerformanceConfigConvBiasActivAsm1x1U ConvBiasActivAsm1x1U::Search(const FusionC
                                                                    const FusionDescription& problem,
                                                                    const AnyInvokeParams&) const
 {
-    auto conv_problem    = problem.GetConvProblem(0, conv::Direction::Forward);
-    conv_problem.bias    = 1;
-    conv_problem.bias_sz = static_cast<size_t>(conv_problem.GetOutChannels()) *
-                           ((conv_problem.GetOutDataType() == miopenHalf) ? 2 : 4);
-    const auto conv_ctx = context.GetConvContext(conv_problem);
-
-    if(!conv_problem.direction.IsForward())
-        MIOPEN_THROW("Only inference supported.");
-
+    AnyInvokeParams fused_invoker;
+    miopen::OperatorArgs params;
     /// Workaround: Fused conv API does not pass user-allocated buffers here,
     /// but we need these buffers for search.
-    auto& handle = conv_ctx.GetStream();
-
-    const auto bias_buf = handle.Create(conv_problem.GetBiasSize());
-    const auto in_buf   = handle.Create(conv_problem.GetInSize());
-    const auto wei_buf  = handle.Create(conv_problem.GetWeightsSize());
-    const auto out_buf  = handle.Create(conv_problem.GetOutSize());
+    AllocateConvBiasActivFusionInvokerBuffer(context, problem, params, fused_invoker);
```

Comment:
For the changes suggested above:
```suggestion
    const auto [params, invoke_params] = AllocateConvBiasActivFusionInvokerBuffer(context, problem);
```

Reply:
I am not sure if this works. Since params is basically vector of `unique_ptr` the same `unique_ptr` is also sent to FusionInvokeParams's constructor. When params is moved during return params that was passed to FusionInvokeParams gets undefined behavior. This eventually cause segfault.

Reply:
Two `unique_ptr`s should never have the same pointer owned. You should use `shared_ptr` if there are two or more owners. And `FusionInvokeParams` should then contain a copy of `OperatorArgs` rather than a reference. This would be much safer to use.
What I want to say is that `FusionInvokeParams` should have some form of explicit ownership over `OperatorArgs`. This may be alternatively conveyed by passing `OperatorArgs` around as shared/weak pointers if later changes are supposed to be reflected in invoke params while they are still owned separately.
But it may be fixed in a separate PR since it's not been implemented in this PR anyway.